### PR TITLE
Update the filename for GCD proof

### PR DESCRIPTION
**Pull request by jonaprieto**
The file  `LTC-PCF/Program/GCD/Partial/ProofSpecification.agda` does not exist.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ the following commands:
 
 ````bash
 $ cd fotc/src/fot
-$ agda LTC-PCF/Program/GCD/Partial/ProofSpecification.agda
+$ agda LTC-PCF/Program/GCD/Partial/GCD.agda
 ````
```
Comment (asr) [inline README.md:88]: The right file is `LTC-PCF.Program.GCD.Partial.CorrectnessProof.agda`. Please fix the PR.

Comment (asr) [reply README.md:88]: Oops, I meant `LTC-PCF/Program/GCD/Partial/CorrectnessProof.agda`.

Comment (asr) [reply README.md:88]: Please fix the PR and I'll merge a squashed commit.

Comment (jonaprieto) [reply README.md:88]: I was my mistake too.

**Comment by asr** [comment]
Thanks!

**Pull request merged**